bitcash/network/services.py

```diff
@@ -32,20 +35,46 @@ def get_balance(cls, address):
 
     @classmethod
     def get_transactions(cls, address):
-        r = requests.get(cls.MAIN_ADDRESS_API + address, timeout=DEFAULT_TIMEOUT)
+        r = requests.get(cls.MAIN_ADDRESS_API.format(address), timeout=DEFAULT_TIMEOUT)
         if r.status_code != 200:  # pragma: no cover
             raise ConnectionError
         return r.json()['transactions']
 
     @classmethod
-    def get_tx_amount(cls, txid, txindex):
-        endpoint = cls.MAIN_TX_AMOUNT_API
+    def get_transaction(cls, txid):
+        r = requests.get(cls.MAIN_TX_API.format(txid), timeout=DEFAULT_TIMEOUT)
+        if r.status_code != 200:  # pragma: no cover
+            raise ConnectionError
+        response = r.json(parse_float=Decimal)
+
+        tx = Transaction(response['txid'], response['blockheight'],
+                (Decimal(response['valueIn'])*100000000).normalize(),
```

Comment:
Let's use a satoshi constant in these. Would be so easy to have a missing zero and not know.